src/view/nftPlugin/cardNFT/imageNFT.tsx

```diff
@@ -1,23 +1,14 @@
-import { Image } from 'antd'
+import { Avatar } from 'antd'
 
-import IMAGE_DEFAULT from 'static/images/nft.jpeg'
+import IMAGE_DEFAULT from 'static/images/nft-default.svg'
 
 type ImageNFTProps = {
```

Comment:
always export type as a team convention

Reply:
@tuphan-dn I have fixed. Please check again. Thank you sir!